### PR TITLE
[MU4] Fixed instruments titles after creating a score

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: 
https://github.com/musescore/MuseScore/issues/8842
https://github.com/musescore/MuseScore/issues/8834

### CODE REVIEW

**Review by Eism — APPROVED** [review]

File: src/engraving/libmscore/instrtemplate.cpp [hunk 1]
```diff
@@ -184,7 +184,7 @@ void InstrumentGroup::clear()
 
 InstrumentTemplate::InstrumentTemplate()
 {
-    staves             = 1;
+    stavesCount        = 1;
```
Comment (Jojo-Schmitz) [inline src/engraving/libmscore/instrtemplate.cpp:187]: Isn't `staffCount` more correct.

Comment (RomanPudashkin) [reply src/engraving/libmscore/instrtemplate.cpp]: fixed

**Comment by vshalkevich** [comment]
Fixed. Works fine

**Comment by vshalkevich** [comment]
2021-08-12 19_15_56-Untitled
rash if click on Create new score

**Pull request merged**